twolame.pc.in

```diff
@@ -8,4 +8,4 @@ Description: An optimized MPEG Audio Layer 2 encoder
 Requires:
 Version: @VERSION@
 Libs: -L${libdir} -ltwolame
-Cflags: -I${includedir}
+Cflags:  @CFLAGS@ -I${includedir}
```

Comment:
I am not sure your local current local CFLAGS should be put into twolame.pc?
What problem were you trying to solve by doing this?